ansible-ipi-install/README.md

```diff
@@ -279,6 +279,12 @@ dnsvip=""
 # Network Type (OpenShiftSDN or OVNKubernetes). Playbook defaults to OVNKubernetes.
 # Uncomment below for OpenShiftSDN
 #network_type="OpenShiftSDN"
+# A URL to override the default operating system image for the bootstrap node.
```

Comment:
# (Optional) A URL to override the default operating system image for the bootstrap node.

Reply:
Done, thanks!